src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -291,6 +291,8 @@ exprt gdb_value_extractort::get_pointer_to_member_value(
   bool found = false;
   CHECK_RETURN(maybe_struct_size.has_value());
   for(const auto &value_pair : values)
+
+  if(memory_map.count(struct_name) == 0)
```

Comment:
This looks weird.

Reply:
Wrapped in a helper function `has_known_memory_location`.